common/dagmcmetadata.cpp

```diff
@@ -183,11 +183,14 @@ void dagmcMetaData::parse_material_data()
     } else {
       grp_name = "mat:"+material_props[0];
       volume_density_data_eh[eh] = "";
-      if(try_to_make_int(material_props[0])){
-	std::cout << "Using the simplified nameing scheme without a density" << std::endl;
-	std::cout << "property is forbidden, please rename the group mat:" << material_props[0] << std::endl;
-	exit(EXIT_FAILURE);
-      }
+    }
+
+    // check to see if the simplied naming scheme is used, by try to convert the
+    // material property to an int
+    if(try_to_make_int(material_props[0]) && !(DAG->is_implicit_complement(eh))){
```

Comment:
I think this is breaking things... if you add a volume to the group `mat:1/rho:-12.3` and then run MCNP, you get

```
Using the simplified nameing scheme without a density
property is forbidden, please rename the group mat:1
```